src/ConnectedRouter.js

```diff
@@ -2,6 +2,7 @@ import React, { PureComponent } from 'react'
 import PropTypes from 'prop-types'
 import { connect, ReactReduxContext } from 'react-redux'
 import { Router } from 'react-router'
+import { isEqual } from 'lodash'
```

Comment:
`import isEqual from 'lodash.isequal'` to avoid loading the whole lodash module. 

Please run `yarn add -P lodash.isequal` to update package.json and yarn.lock as peerDependencies.